Userland/Libraries/LibWebView/InspectorClient.cpp

```diff
@@ -523,11 +523,13 @@ String InspectorClient::generate_dom_tree(JsonObject const& dom_tree)
 
         if (auto attributes = node.get_object("attributes"sv); attributes.has_value()) {
             attributes->for_each_member([&](auto const& name, auto const& value) {
+                auto value_string = value.as_string_or({});
```

Comment:
Hmm, it is probably better to use `JsonValue::as_string` and not hide type mismatch.

Looks like my oversight that `as_string_or` still exists :)